### PR TITLE
Issue326

### PR DESCRIPTION
**Pull request by George-Onyango**
# Description

This pull request adds a feature to the application

Fixes # (issue) - (#326 )

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

# Screenshot (optional)

# Web
Screenshot from 2023-02-27 23-22-17

# Mobile
Screenshot from 2023-02-27 23-22-36

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. 
Please also list any relevant details for your test configuration if necessary.
- [x] Test responsiveness with different screen sizes

# Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: libs/features/components/banners/src/lib/banners/elewa-group-brands/elewa-group-brands/elewa-group-brands.component.ts [hunk 1]
```diff
@@ -7,10 +7,10 @@ import { Component, Input } from '@angular/core';
...
-  @Input () logo: string = 'https://res.cloudinary.com/dyl3rncv3/image/upload/v1675690295/elewa-group-website/Icons/SVG/Elewa_education_pp1gnc.svg';
-  @Input () name: string = '';
-  @Input () brandDescription: string = ''
-  @Input () buttonText: string = ''
+  @Input () logo = 'https://res.cloudinary.com/dyl3rncv3/image/upload/v1675690295/elewa-group-website/Icons/SVG/Elewa_education_pp1gnc.svg';
+  @Input () name = '';
+  @Input () brandDescription = ''
+  @Input () buttonText = ''
   @Input() url = '';
   @Input () logoPlacement: 'right' | 'left' = 'right';
```
Comment (IanOdhiambo9) [inline libs/features/components/banners/src/lib/banners/elewa-group-brands/elewa-group-brands/elewa-group-brands.component.ts:15]: Why are you removing the types?

File: libs/pages/elewa/brands/src/lib/pages-elewa-brands.module.ts [hunk 2]
```diff
@@ -9,9 +9,14 @@ import { ElewaBrandsOpportunitiesComponent } from './components/elewa-brands-opp
 import { ElewaBrandsPageComponent } from './pages/elewa-brands-page/elewa-brands-page/elewa-brands-page.component';
 
 import { BrandsRoutingModule } from './brands.routing';
+import { BrandsCreativeHubComponent } from './components/brands-creative-hub/brands-creative-hub.component';
```
Comment (IanOdhiambo9) [inline libs/pages/elewa/brands/src/lib/pages-elewa-brands.module.ts:12]: order of imports?

**Review by IanOdhiambo9 — CHANGES_REQUESTED** [review]

File: libs/features/components/banners/src/lib/banners/elewa-group-brands/elewa-group-brands/elewa-group-brands.component.ts (hunk 1 above)
Comment (George-Onyango) [reply libs/features/components/banners/src/lib/banners/elewa-group-brands/elewa-group-brands/elewa-group-brands.component.ts:15]: I sent you a screenshot explaining the reason it was not passing values when the types have been defined then the string comes after
I shall make changes accordingly

**Comment by George-Onyango** [comment]
@IanOdhiambo9  kindly review

**Pull request merged**